Services/Clustering/DevicePartitions.cs

```diff
@@ -73,6 +88,27 @@ public async Task CreateAsync(string simulationId)
             this.log.Debug("Partitions created", () => new { simulationId });
         }
 
+        // Return partitions ready to be simulated, not yet assigned to any node.
+        public async Task<IList<DevicesPartition>> GetUnassignedAsync(string simulationId)
+        {
+            var nodeId = this.clusterNodes.GetCurrentNodeId();
+
+            this.log.Debug("Searching partitions not assigned to any node...", () => new { simulationId, nodeId });
+
+            var partitions = await this.partitionsStorage.GetAllAsync();
+
+            var result = partitions
+                .Where(p => !p.IsLocked())
+                .Select(p => JsonConvert.DeserializeObject<DevicesPartition>(p.Data))
+                .Where(x => x.SimulationId == simulationId)
+                .ToList();
+
```

Comment:
nit: fix empty lines in this function and below function. Blank lines are added in this function but not present in the function below - "TryToAssignPartitionAsync"

Reply:
Fixed.